src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IDelegateCreationExpression.cs

```diff
@@ -159,11 +159,13 @@ void Main()
             string expectedOperationTree = @"
 IDelegateCreationOperation (OperationKind.DelegateCreation, Type: System.Action) (Syntax: '(Action)(() => { })')
   Target: 
-    IAnonymousFunctionOperation (Symbol: lambda expression) (OperationKind.AnonymousFunction, Type: null) (Syntax: '() => { }')
-      IBlockOperation (1 statements) (OperationKind.Block, Type: null) (Syntax: '{ }')
-        IReturnOperation (OperationKind.Return, Type: null, IsImplicit) (Syntax: '{ }')
-          ReturnedValue: 
-            null
+    IParenthesizedOperation (OperationKind.Parenthesized, Type: null) (Syntax: '(() => { })')
```

Comment:
We need to update the comment on IDelegateCreationExpression.Target to indicate IParenthesizedOperation is a valid node to expect.

Reply:
Actually, I'm not sure that we should be generating an IParenthesizedOperation here. This is equivalent to `DirectCast(Sub() End Sub, Action)` in VB, and there's no IParenthesizedOperation there.

---
In reply to: [148647044](https://github.com/dotnet/roslyn/pull/22846#discussion_r148647044) [](ancestors = 148647044)

Reply:
Found and filed [https://github.com/dotnet/roslyn/issues/22988](https://github.com/dotnet/roslyn/issues/22988), for `CType((Sub() End Sub), Action)`.

---
In reply to: [148648204](https://github.com/dotnet/roslyn/pull/22846#discussion_r148648204) [](ancestors = 148648204,148647044)

Reply:
So, we need to update it regardless, as the fix for the above bug will have `IParenthesizedOperation` be a valid option for the target. However, I still feel like, for the first set of parens, we should try to match the VB tree. IE: `(Action)(() => {})` == `CType(Sub() End Sub, Action)`, `(Action)((() => {}))` == `CType((Sub() End Sub), Action)`.

Reply:
Seems like a valid point, lets talk in person today to discuss